packages/qwik/src/cli/check-client/constant.ts

```diff
@@ -0,0 +1,6 @@
+import path from 'path';
+
+export const DISK_DIR: string = path.resolve('dist');
+export const SRC_DIR: string = path.resolve('src');
```

Comment:
actually, let's get the source dir from the command line as well, so these can go too